database/index.js

```diff
@@ -0,0 +1,44 @@
+//in here set up mongoose database
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+
+// var bodyParse = require('body-parser');
+mongoose.connect('mongodb://localhost/helmets', { useNewUrlParser: true });
+
+//this is the piece I was missing to be able to use the mongoose connect methods.
+var db = mongoose.connection;
+
+var dropDB = function() {
+  //access the DB as an array
+  db.collections['helmets'].drop(function(err) {
+    console.log('HELMETS DROPPED');
+  });
+};
+
+var helmetsSchema = new Schema({
```

Comment:
`helmetSchema` might be a better name, as each item represented is a single helmet